nion/swift/test/Inspector_test.py

```diff
@@ -1440,12 +1440,13 @@ def test_setting_display_layer_to_no_display_data_channel_and_back(self):
             display_panel.set_display_panel_display_item(display_item)
             document_controller.periodic()
             inspector_panel = document_controller.find_dock_panel("inspector-panel")
-            display_data_channel_index_widget = inspector_panel.widget.find_widget_by_id("display_data_channel_index_widget")
-            display_data_channel_index_widget.on_editing_finished(None)
-            display_data_channel_index_widget.on_editing_finished(0)
-            display_data_channel_index_widget.on_editing_finished("")
-            display_data_channel_index_widget.on_editing_finished("A")
-            display_data_channel_index_widget.on_editing_finished(0)
+            line_plot_display_layers_section = inspector_panel.widget.find_widget_by_id("line_plot_display_layers_inspector_section")
+            line_plot_display_layer_model = line_plot_display_layers_section._handler._line_plot_display_layer_handlers[0]._line_plot_display_layer_model
+            line_plot_display_layer_model.data_index_model.Value = None
+            line_plot_display_layer_model.data_index_model.Value = 0
+            line_plot_display_layer_model.data_index_model.Value = ""
+            line_plot_display_layer_model.data_index_model.Value = "A"
+            line_plot_display_layer_model.data_index_model.Value = 0
```

Comment:
Why do you test =0 twice?

Reply:
Yes, the particular test is `test_setting_display_layer_to_no_display_data_channel_and_back`, the last set to 0 is the `back` of that.